### PR TITLE
Convert module to be Fork 5 compatible.

### PR DESCRIPTION
**Pull request by ohvitorino**
The module is currently not compatible with Fork 5. This PR makes the necessary changes to make it compatible.

### CODE REVIEW

File: src/Backend/Modules/Partners/Actions/Delete.php [hunk 1]
```diff
@@ -9,18 +9,25 @@
...
         if (!$widget instanceof Widget) {
-            return $this->redirect(Model::createURLForAction('Index', null, null, ['error' => 'non-existing']));
+            $this->redirect(
+                Model::createURLForAction(
+                    'Index',
+                    null,
+                    null,
+                    ['error' => 'non-existing']
+                )
+            );
```
Comment (jonasdekeukelaere) [inline src/Backend/Modules/Partners/Actions/Delete.php:26]: Best hier nog een `return;` zetten achter de redirect.

File: src/Backend/Modules/Partners/Actions/Edit.php [hunk 2]
```diff
@@ -10,24 +10,24 @@
...
     {
         parent::execute();
         $widget = $this->get('partners.repository.widget')->find(
-            $this->getParameter('id', 'int')
+            $this->getRequest()->query->getInt('id')
         );
 
         if (!$widget instanceof Widget) {
-            return $this->redirect(Model::createURLForAction('Index', null, null, ['error' => 'non-existing']));
+            $this->redirect(Model::createURLForAction('Index', null, null, ['error' => 'non-existing']));
```
Comment (jonasdekeukelaere) [inline src/Backend/Modules/Partners/Actions/Edit.php:21]: same

**Review by jonasdekeukelaere — CHANGES_REQUESTED** [review]

File: src/Backend/Modules/Partners/Actions/Delete.php (hunk 1 above)
Comment (ohvitorino) [reply src/Backend/Modules/Partners/Actions/Delete.php:26]: Fixed 1a10186f

File: src/Backend/Modules/Partners/Actions/Edit.php (hunk 2 above)
Comment (ohvitorino) [reply src/Backend/Modules/Partners/Actions/Edit.php:21]: Fixed 1a10186f

**Review by StijnVrolijk — APPROVED** [review]

**Review by jonasdekeukelaere — APPROVED** [review]

**Pull request merged**